js/preview/src/build.ts

```diff
@@ -43,23 +43,25 @@ export async function make_build({
 							emptyOutDir: true,
 							outDir: join(template_dir, entry),
 							lib: {
-								entry: join(source_dir, entry, "index.ts"),
+								entry: join(source_dir, path),
 								fileName: "index.js",
 								formats: ["es"]
 							},
+							minify: true,
 							rollupOptions: {
 								output: {
-									entryFileNames: "[name].js"
+									entryFileNames: (chunkInfo: PreRenderedChunk) =>
+										`${chunkInfo.name.toLocaleLowerCase()}.js`
 								}
 							}
 						}
 					});
 				} catch (e) {
-					console.error(e);
+					throw e;
 				}
 			}
 		}
 	} catch (e) {
-		console.error(e);
+		throw e;
```

Comment:
We were squashing errors in the try/catch which is why this didn't error out as it should have. This will now fail noisily if the build doesn't succeed.